lib/crypto_bottom_sheet/view/crypto_bottom_sheet_view.dart

```diff
@@ -50,42 +51,71 @@ class _CryptoBottomSheetPageState extends State<CryptoBottomSheetPage> {
     );
 
     if (newCryptoAccountName != null &&
-        newCryptoAccountName != cryptoAccountData.name) {
-      await context.read<CryptoBottomSheetCubit>().editCryptoAccount(
-            newAccountName: newCryptoAccountName,
-            index: index,
-          );
+        newCryptoAccountName != cryptoAccountData.name &&
+        newCryptoAccountName.isNotEmpty) {
+      if (accountNameList.contains(newCryptoAccountName)) {
```

Comment:
Users can put any name.. Do not restrict users from putting the same name...The user needs freedom..